### DISCUSSION TITLE
[功能请求]: 添加值日表功能

### DISCUSSION THREAD

**Discussion by gaopx11233** [ClassIsland/ClassIsland#556] [插件功能请求]
### 检查清单

- [x] 我已经在 [Issues](https://github.com/HelloWRC/ClassIsland/issues?q=) 中找过我要提出的问题，没有找到相同问题的ISSUE。
- [X] 我已经详细阅读过应用的帮助文档，但我想要的功能没有实现。
- [X] 我已确认我的功能尚未在 [dev](https://github.com/HelloWRC/ClassIsland/commits/dev/) 分支实现。

### 背景与动机

添加值日表功能并自动显示可以减小生活委员的工作量

### 想要实现或优化的功能

添加值日表功能

**Comment by LiPolymer** [comment]
🥳 该功能已在[ExtraIsland](https://github.com/LiPolymer/ExtraIsland/)R0.30初步实现(已经上架插件市场)
欢迎下载! 如果觉得有用,请星标仓库并把本条回复标记为答案,谢谢!😋
